virt/virtual_machines/advanced_vm_management/virt-using-containerdisk-volumes.adoc

```diff
@@ -0,0 +1,18 @@
+[id="virt-using-containerdisk-volumes"]
+= Using containerDisk volumes with virtual machines
+include::modules/virt-document-attributes.adoc[]
+:context: virt-using-containerdisk-volumes
+toc::[]
+
```

Comment:
I'd appreciate if we can put a disclaimer here that the use-case for containerDisks is narrow, and that the general recommendation is to use CDI et al for VM storage.

Reply:
I think the updated concept (modules/virt-about-container-disks.adoc) satisfies this request now